source/frontend/src/components/layout/SideNavBar/SubdCons.tsx

```diff
@@ -0,0 +1,39 @@
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+
+import { Claims } from '@/constants/claims';
+import useKeycloakWrapper from '@/hooks/useKeycloakWrapper';
+
+import { ISideTrayPageProps } from './SideTray';
+import * as Styled from './styles';
+
+/**
+ * Subdivision & Consolidation Tray page.
+ * Intended for use within the left side tray.
+ */
+export const SubdivisionConsolidationTray = ({ onLinkClick }: ISideTrayPageProps) => {
+  const { hasClaim } = useKeycloakWrapper();
+  return (
+    <>
+      <HalfHeightDiv>
+        <Styled.TrayHeader>Disposition Files</Styled.TrayHeader>
```

Comment:
not sure this header is correct? I would expect it to be "Consolidation & Subdivision" - not disposition file,,,

Reply:
good point. I will update it in the front end pr